WSL/troubleshooting.md

```diff
@@ -239,6 +239,70 @@ Once you have disconnected the VPN, you will have to revert the changes to `/etc
 2. `sudo mv resolv.conf resolv.conf.new`
 3. `sudo ln -s ../run/resolvconf/resolv.conf resolv.conf`
 
+### Cisco Anyconnect VPN issues with WSL2 in NAT mode
+
+The Cisco AnyConnect VPN modifies routes in a way which prevents NAT from working. There are two potential workarounds: Cisco has some explicit documentation for WSL2 - it is likely the same thing that is needed here. [Cisco AnyConnect Secure Mobility Client Administrator Guide, Release 4.10 - Troubleshoot AnyConnect [Cisco AnyConnect Secure Mobility Client] - Cisco](https://www.cisco.com/c/en/us/td/docs/security/vpn_client/anyconnect/anyconnect410/administration/guide/b-anyconnect-admin-guide-4-10/troubleshoot-anyconnect.html#Cisco_Task_in_List_GUI.dita_3a9a8101-f034-4e9b-b24a-486ee47b5e9f)
+
+### WSL connectivity issues with VPNs when Mirrored mode is on
+VPNs that we tested and confirmed are incompatible with WSL.
+1.	Bitdefender VPN – version 26.0.2.1
+2.	OpenVPN – version 2.6.501
+3.	Mcafee Safe Connect – version 2.16.1.124
+
+### Setting up HttpProxy Mirroring in WSL2
+HTTP/S proxy mirroring is enabled by default. It can be disabled setting “AutoProxy=False” in the .wslconfig file. If the proxy settings are changed, the WSL instance must be restarted for the proxy settings to take effect. Note: A toast will be displayed to the user telling them that the WSL instance must be restarted for the new settings to take effect.
```

Comment:
nit: Can you use the " character instead of the quote that has been used? (Down below, the "useWindowsDnsCache" uses the proper character.